pep-9999.rst

```diff
@@ -0,0 +1,316 @@
+PEP: 9999
+Title: The manylinux2014 Platform Tag
+Version: $Revision$
+Last-Modified: $Date$
+Author: Dustin Ingram <di@python.org>
+BDFL-Delegate: Paul Moore <p.f.moore@gmail.com>
+Discussions-To: Distutils SIG <distutils-sig@python.org>
```

Comment:
To reflect the current discussion, should this instead be https://discuss.python.org/t/the-next-manylinux-specification/ ?

Reply:
Possibly, I'm not sure what the status quo is here these days.

Reply:
I think it should reflect the reality, which is that we're using Discourse.